src/core/annotation.js

```diff
@@ -193,13 +243,9 @@ var Annotation = (function AnnotationClosure() {
 
     isInvisible: function Annotation_isInvisible() {
       var data = this.data;
-      if (data && SUPPORTED_TYPES.indexOf(data.subtype) !== -1) {
-        return false;
-      } else {
-        return !!(data &&
-                  data.annotationFlags &&            // Default: not invisible
-                  data.annotationFlags & 0x1);       // Invisible
-      }
+      return !!(data &&
+                data.annotationFlags &&            // Default: not invisible
+                data.annotationFlags & 0x1);       // Invisible
```

Comment:
Funny that you mention that, as that is the next item on my list of refactoring work for the annotation layer! It is already part of my previous PR at https://github.com/mozilla/pdf.js/pull/5218/files#diff-18ca06e2bd5be4c9132b51f78388b8f0R339, but I will make that a loose PR soon. (All my previous PRs are actually mostly parts of that one PR.)